code/game/objects/structures/crates_lockers/closets/secure/freezer.dm

```diff
@@ -152,8 +152,14 @@
 		return
 	cooldown = 5
 	var/containers = 0
+	var/containers_ready = FALSE
 	for(var/obj/item/reagent_container/glass/I in freezer.contents)
 		if(I.reagents.replace_with(polymerization_recipe, "paraformaldehyde", 3))
 			containers++
+			if(!I.reagents.has_reagent("formaldehyde", 3) || !I.reagents.has_reagent("water", 3))
+				containers_ready = TRUE
 		if(containers > 3)
 			break
+
+	if(containers_ready) //at least 1 container has finished, ring the bell
+		playsound(freezer.loc, 'sound/misc/desk_bell.ogg', 150)
```

Comment:
I think ding.ogg in machines would be better since its a machine finishing rather than a person ringing a desk bell.

Reply:
done